crates/primitives/block/src/header.rs

```diff
@@ -101,7 +106,6 @@ impl Header {
             self.event_count.into(),
             self.event_commitment.into(),
             self.protocol_version.into(),
-            Felt252Wrapper::ZERO,
```

Comment:
https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/header/#block_hash

Are you sure about that?
Can you link the source you use as a reference for this impl?

Reply:
My apologies this is a mistake I did when correcting the corresponding tests. Added back.

Reply:
you added it back in the last position when it was in the penultimate one before.